__tests__/app.test.js

```diff
@@ -0,0 +1,173 @@
+  test('Update article so the vote counts increases by "newVote" property (positive number)', () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.votes).toBe(101);
+      });
+  });
+  test('Update article so the vote counts decrease by "newVote" property (negative number)', () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -10 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.votes).toBe(90);
+      });
+  });
+  test("status 400: Bad request. Returns a 400 error when passed the wrong data type into inc_votes object", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "not a number" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad Request");
+      });
+  });
+});
+describe("GET api/users", () => {
+  test("should return an array of objects containing the username property", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.users).toHaveLength(4);
+        response.body.users.forEach((user) => {
+          expect(user).toEqual(
+            expect.objectContaining({
+              username: expect.any(String),
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+});
+describe("GET api/articles", () => {
+  test("should return an array of articles containing the article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles).toHaveLength(12);
+        response.body.articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+});
+describe("GET /api/articles/:article:id (comment count)", () => {
+  test("Article response object should now include the comment count", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: expect.any(String),
+            votes: 100,
+            comment_count: expect.any(Number),
```

Comment:
it's not any number, you should assert how many comments you expect this to have

Reply:
also you should need to test the whole object again just this property

Reply:
specific comment count added
object properties removed bar comment_count